### DISCUSSION TITLE
Any plans on a PS2 port?

### DISCUSSION THREAD

**Discussion by boywifee** [nzp-team/nzportable#940] [General]
I have my old PS2 laying around. Was wondering if there were any sort of plans on porting this game onto the PS2. It already exists for the PS Vita, 3DS and PSP so I don't see why not the PS2.

**Comment by Peter0x44** [comment]
There was discussion about this prior, the ps2 does not have any decent quake source ports capable of supporting our needs. So it likely won't happen.

**Reply by boywifee** [reply]
aw sucks

**Comment by MotoLegacy** [comment]
To add to this, the Emotion Engine is absolutely dependent on asynchronous tasking which NZP Engine is architecturally inadequate for. The NZP engine would have to be completely refactored just to accommodate this port.

**Reply by boywifee** [reply]
damn, would've been cool though

**Comment by ProfessorZIRE** [comment]
If possible u could make it via a half life mod cos it has a ps2 ver

**Reply by MotoLegacy** [reply]
nzp isnt halflife, its an entirely custom engine

**Reply by ProfessorZIRE** [reply]
I know, I'm saying that it still may be possible from someone's knowledge on the half-life engine

**Reply by MotoLegacy** [reply]
no.
